archimedes/archimedes/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.0"
```

Comment:
When we package archimedes and release it as a pypi package, this will be our version.  Currently not prioritizing that, though.